mobile_net/utils/dataset.py

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
+
+"""
+Loads all images into cache and create a tf.Dataset Object
+"""
+import numpy as np
+import os
+import tensorflow as tf
+import cv2
+import tqdm
+
+
+class Dataset:
```

Comment:
… based on my above-written doubts that we make this into a library in the end we may not need a class but instead just write the preprocessing pipeline at the start of the script that sets up, trains and exports the model?